bakery/rb_1_with_table_guide.py

```diff
@@ -0,0 +1,78 @@
+import pandas as pd
```

Comment:
what is the purpose of this file? Is it a sort of library for 1 qubit RB? If so let's create a python file which is really a 1 qubit RB library which allows the user to import everything he needs in order to run 1 qubit RB in an easy manner. This then should not include functions like "measure state" which are not related to the RB itself.